packages/prisma/seed-app-store.ts

```diff
@@ -245,6 +245,12 @@ export default async function main() {
       consumer_secret: process.env.SALESFORCE_CONSUMER_SECRET,
     });
   }
+  if (process.env.ZOHOCRM_CLIENT_ID && process.env.ZOHOCRM_CLIENT_SECRET) {
+    await createApp("zohocrm", "zohocrm", ["other"], "zohocrm_other_calendar", {
```

Comment:
This seeding is a duplicate of that in seed-app-store.config.json(which is automatically generated).

The idea of seed-app-store.config.json to avoid this manual code addition for an app, but seed-app-store.config.json doesn't support this conditional seeding based on env variables. 
We should make App aware of requirement of env variables and app-store-cli should enable creation of such an app.